pkg/conversion/unversioned_test.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conversion_test
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/testapi"
```

Comment:
I mean like make

```
pkg/conversion/internal/test/apis/v1/types.go
pkg/conversion/internal/test/apis/unversioned/types.go
```

And then add just enough code to those packages to use them in the tests, instead of using the real pkg/api & pkg/apis/unversioned packages.

But just add a TODO-- we've got too much to do to do this right now. :)